src/definitions/collections/breadcrumb.less

```diff
@@ -12,8 +12,8 @@
             Theme
 *******************************/
 
-@type : 'collection';
-@element : 'breadcrumb';
+@type: 'collection';
```

Comment:
sadly not fixed by `declaration-colon-space-before` rule (but enabled in https://github.com/stylelint/stylelint-config-standard/blob/29.0.0/index.js#L76)